### PR TITLE
fix: building custom apps when using unix-based os and not pnpm

### PR DESCRIPTION
**Pull request by SunsetTechuila**
fixes building via gh actions

### CODE REVIEW

**Comment by SunsetTechuila** [comment]
hope this is the last one

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .vscode
-node_modules
\ No newline at end of file
+node_modules
+dist
```
Comment (Pithaya) [inline .gitignore:3]:
```suggestion
custom-apps/*/dist
```
The dist folder should be kept for the extensions

**Review by Pithaya — APPROVED** [review]

**Comment by SunsetTechuila** [comment]
@Pithaya you need to manually trigger the workflows now

**Comment by Pithaya** [comment]
@SunsetTechuila Done!

**Comment by SunsetTechuila** [comment]
yes, fixed
https://github.com/Pithaya/spicetify-apps-dist/blob/dist/eternal-jukebox/index.js

**Pull request merged**